splinepy/microstructure/tiles/crosstile3d.py

```diff
@@ -63,6 +65,11 @@ def closing_tile(
                 * 0.2
             )
 
+        if parameter_sensitivities is not None:
+            raise NotImplementedError(
```

Comment:
will this be a next PR or next next ... next PR?

Reply:
"As required"